file/logfiles.py

```diff
@@ -72,12 +72,13 @@ def save_syslog(self):
     def save_system_log(self):
         # save system logs
         if self.options.syslog:
-            if util.get_init_type() == "systemd":
+            if util.get_init_type() == "systemd" and self.options.systemd:
```

Comment:
and or or?

Reply:
`and`, the new logic is only collect journald log when `--systemd` is set